From ba720c11041f65fd34e6b814899d3fa8466e8867 Mon Sep 17 00:00:00 2001
From: Nirbheek Chauhan <nirbheek@centricular.com>
Date: Fri, 21 Jan 2022 15:34:21 +0530
Subject: [PATCH] cmake: Don't name MSVC DLL 'wavpackdll.dll'

It breaks the pkgconfig file and is very unexpected overall. Probably
can't be upstreamed.
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7cd940c..970505b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -229,7 +229,7 @@ endif()
 
 if(WIN32 AND (NOT MINGW))
     if(BUILD_SHARED_LIBS)
-        set_target_properties(wavpack PROPERTIES OUTPUT_NAME wavpackdll)
+        set_target_properties(wavpack PROPERTIES OUTPUT_NAME wavpack)
     else()
         set_target_properties(wavpack PROPERTIES OUTPUT_NAME libwavpack)
     endif()
-- 
2.34.1.windows.1

